src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1045,6 +1046,44 @@ public bool EnableEditing
             }
         }
 
+        /// <summary>
+        /// Specify horizontal scroll position in text control.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int HorizontalScrollPosition
+        {
+            get
+            {
+                int temp;
+                GetProperty(TextEditor.Property.HORIZONTAL_SCROLL_POSITION).Get(out temp);
```

Comment:
I want to suggest to fix this code for avoiding CA2000 build warnings: "Call System.IDisposable.Dispose on object created by ~~"

Just like:
                using (PropertyValue propertyValue = GetProperty(TextEditor.Property.LINE_COUNT))
                {
                    propertyValue.Get(out temp);
                }
                
                return temp;

or
just call Dispose or Close at end of code.